datanode/gateway/graphql/schema.graphql

```diff
@@ -2279,6 +2286,63 @@ type Order {
   liquidityProvision: LiquidityProvision
 }
 
+"An order update in Vega, if active it will be on the order book for the market"
+type OrderUpdate {
+  "Hash of the order data"
+  id: ID!
+
+  "The worst price the order will trade at (e.g. buy for price or less, sell for price or more) (uint64)"
+  price: String!
+
+  "The timeInForce of order (determines how and if it executes, and whether it persists on the book)"
+  timeInForce: OrderTimeInForce!
+
+  "Whether the order is to buy or sell"
+  side: Side!
+
+  "The market the order is trading on (probably stored internally as a hash of the market details)"
+  marketId: ID!
+
+  "Total number of contracts that may be bought or sold (immutable) (uint64)"
```

Comment:
```suggestion
  "Total number of units that may be bought or sold (immutable) (uint64)"
```

Reply:
What's the issue with contracts here?

Reply:
Contracts implies a legal agreement, it's a word to avoid if possible, at least in descriptions.  Also can't really have a fractional contract so it doesn't make sense for fractional order sizes.